Language/Haskell/GhcMod/Internal.hs

```diff
@@ -35,14 +45,33 @@ module Language.Haskell.GhcMod.Internal (
   , World
   , getCurrentWorld
   , didWorldChange
+  -- -- * Cabal Helper
+  , getComponents
+  , resolvedComponentsCache
+  , resolveEntrypoint
+  , cached
```

Comment:
I'd just like a comment about why you think exporting this function is a good idea?